api/src/config.ts

```diff
@@ -1,9 +1,29 @@
-const getEnvString = (name: string): string => process.env[name] || '';
-const getEnvNumber = (name: string): number => parseInt(getEnvString(name), 10);
-const getEnvBoolean = (name: string): boolean => getEnvString(name) === 'true';
+const getEnvString = (name: string, defaultValue?: string): string => {
+    let value = '';
+    try {
+        if (!(process.env[name]) && (defaultValue === undefined || defaultValue === '')) {
+            throw new Error('Environment variable is undefined');
+        } else if (!(process.env[name])) {
+            value = defaultValue as string;
+        } else {
+            value = process.env[name] as string;
+        }
+    } catch (e) {
+        // tslint:disable-next-line no-console
+        console.error(e);
+    }
+    return value;
+};
+const getEnvNumber = (name: string, defaultValue?: string): number => parseInt(getEnvString(name, defaultValue), 10);
+const getEnvBoolean = (name: string, defaultValue?: string): boolean => {
+    if (getEnvString(name, defaultValue) === 'true' || getEnvString(name, defaultValue) === 'True') {
+        return true;
+    }
+    return false;
```

Comment:
You should assert that if it's not `"true"` or `"True"` that it's `"false"` or `"False"` - that way you don't spend hours debugging an env var with value `true ` or `TRUE`, etc.